### DISCUSSION TITLE
auto translate langage

### DISCUSSION THREAD

**Discussion by abbasudo** [filamentphp/filament#13012] [Ideas]
Hello,
Every text in the panel gets auto-translated. However TextInput labels and generally labels are not getting auto-translated.
![image](https://github.com/filamentphp/filament/assets/86796762/d6387a40-0324-49de-8bc8-d2b01902045b)

I had to add `->label(trans('Name'))` to each input. Is it a good idea to auto-translate this label?

**Comment by abbasudo** [comment]
Also TextColumn at tables needs to be translated.
![image](https://github.com/filamentphp/filament/assets/86796762/ee8245e4-1f66-498d-b615-78bc3e3c8d5d)

**Comment by dissto** [comment]
https://filamentphp.com/docs/3.x/tables/columns/getting-started#setting-a-label 😁

**Reply by abbasudo** [reply]
Thanks!
